README-Hebrew.md

```diff
@@ -0,0 +1,190 @@
+| filename                                                     | אופציונאלי, לא חובה. מיקום חובה ראשון | מלא מראש את השדה "מיקום סקריפט" בממשק המשתמש.                                                                                   |
+| -nc, --no-chrome                                             | אופציונאלי, לא חובה            | פתח את ממשק המשתמש באמצעות דפדפן ברירת המחדל (שעשוי להיות Chrome). לא ינסה למצוא את כרום.                                         |
+| -nu, --no-ui                                                 | אופציונאלי, לא חובה            | אל תנסה לפתוח את ממשק המשתמש בדפדפן ופשוט תדפיס את הכתובת שבה ניתן לגשת לאפליקציה.                     |
+| -c [CONFIG], --config [CONFIG]                               | אופציונאלי, לא חובה            | ספק קובץ תצורה (JSON) למילוי מראש של ממשק המשתמש. ניתן ליצור אותם בלשונית ההגדרות.                               |
+| -o [PATH], --output-dir [PATH]                               | אופציונאלי, לא חובה            | הגדר את ספריית הפלט המוגדרת כברירת מחדל. עדיין ניתן לשנות זאת בממשק המשתמש.                                                            |
+| -bdo [FOLDER_PATH], --build-directory-override [FOLDER_PATH] | אופציונאלי, לא חובה            | תעקוף את ספריית הבנייה המוגדרת כברירת מחדל. שימושי אם אתה צריך לרשום תיקיה כדי למנוע מהאנטי-וירוס להסיר קבצים.        |
+| -lang [LANGUAGE_CODE], --language [LANGUAGE_CODE]            | אופציונאלי, לא חובה            | רמז לממשק המשתמש לאיזו שפה הוא צריך כברירת מחדל בעת הפתיחה. ניתן למצוא קודי שפה בטבלה תחת "תרגום" למטה. |
+
+### תצורת JSON
+
+במקום להכניס את אותם נתונים לממשק המשתמש שוב ושוב, אתה יכול לייצא את המצב הנוכחי על ידי מעבר לקטע "תצורה" בכרטיסיית ההגדרות וייצוא התצורה לקובץ JSON. לאחר מכן ניתן לייבא את זה שוב לממשק המשתמש כדי לאכלס מחדש את כל השדות.
+
+פעולת ייצוא תצורת JSON זו אינה שומרת את ספריית הפלט באופן אוטומטי מכיוון שהעברת מארחים עשויה להיות מבני ספרייה שונים. אם ברצונך לקבל את ספריית הפלט בתצורת JSON, הוסף את הספרייה תחת `nonPyinstallerOptions.outputDirectory` בקובץ JSON (יצטרך ליצור מפתח חדש).
+
+## סרטון
+
+אם אתה צריך משהו ויזואלי שיעזור לך להתחיל, [הכנתי סרטון עבור הגרסה המקורית של הפרויקט הזה](https://youtu.be/OZSZHmWSOeM); דברים מסוימים עשויים להיות שונים אך אותם מושגים עדיין חלים.
+
+## תורם
+
+עיין ב-[CONTRIBUTING.md](./CONTRIBUTING.md) כדי לראות הנחיות כיצד לתרום. הקובץ מתאר מה לעשות אם יש לך תכונה חדשה, שינוי, עדכון תרגום או שמצאת בעיה עם auto-py-to-exe.
+
+## בעיות בשימוש בכלי
+
+אם אתה נתקל בבעיות עם קובץ ההפעלה הארוז או בשימוש בכלי זה באופן כללי, אני ממליץ לך לקרוא את [הפוסט בבלוג שלי על בעיות נפוצות בעת שימוש ב-auto-py-to-exe](https://nitratine.net/blog/post /issues-when-using-auto-py-to-exe/?utm_source=auto_py_to_exe&utm_medium=readme_link&utm_campaign=auto_py_to_exe_help). פוסט זה מכסה דברים שכדאי לדעת על אריזת סקריפטים של Python ותיקונים לדברים שבדרך כלל משתבשים.
+
+אם אתה סבור שמצאת בעיה בכלי זה, עקוב אחר הסעיף ["דיווח על בעיה" ב-CONTRIBUTING.md](./CONTRIBUTING.md#reporting-an-issue).
+
+## תרגומים
```

Comment:
The translation table can be left out

Reply:
not clear on your meaning. I have translated only the columns' captions, not the table contents. should I change/revert something?

Reply:
Check out the other translated README files - notice they do not have the "Translations" section. This section is only in the README.md.